.github/scripts/tests/tests.sh

```diff
@@ -175,7 +197,17 @@ apply_pip_server_configmap() {
   echo "---------------------------------"
   echo "Apply PIP Server ConfigMap"
   echo "---------------------------------"
-  ( cd "${GIT_WORKSPACE}/.github/resources/pypiserver/base" && kubectl apply -f $RESOURCES_DIR_PYPI/nginx-tls-config.yaml -n $DSPA_NAMESPACE )
+  for ns in $DSPA_NAMESPACE $DSPA_K8S_NAMESPACE; do
```

Comment:
Is this still needed? Is the Kubernetes native API test also testing the custom PyPi server?

Reply:
Yes Matt, I thought that way we can test the scenario where the pipeline was uploaded manually.